webapp/plugins/vue-filters.js

```diff
@@ -45,9 +46,9 @@ export default ({ app = {} }) => {
       if (length <= 0) {
         return value
       }
-      let output = value.substring(0, length)
+      let output = trunc(value, length).html
       if (output.length < value.length) {
-        output += '…'
+        output += ' …'
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-23T10:34:37Z" title="Monday, September 23rd 2019, 12:34:37 pm +02:00">Sep 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -45,9 +46,9 @@ export default ({ app = {} }) => {
       if (length <= 0) {
         return value
       }
-      let output = value.substring(0, length)
+      let output = trunc(value, length).html
       if (output.length < value.length) {
-        output += '…'
+        output += ' …'
```


I really love this @mattwr18 ! 💫😍
The ellipse the most and for sure the space as well. 💥